ner_v2/tests/numeral/number/en/test_number_detection.py

```diff
@@ -1,254 +1,84 @@
 from __future__ import absolute_import
-
+import io
+import os
+import six
+import yaml
 from django.test import TestCase
 
-
-from ner_v2.detectors.numeral.utils import get_number_from_number_word
 from ner_v2.detectors.numeral.number.number_detection import NumberDetector
-from ner_v2.detectors.numeral.number.standard_number_detector import NumberVariant
-
-
-class NumberDetectionTest(TestCase):
```

Comment:
Instead of deleting this, rename this to `TestNumberFromWords` and only keep tests that test `get_number_from_number_word`

Reply:
okay